### DISCUSSION TITLE
Enabling pluings via kubeadm

### DISCUSSION THREAD

**Discussion by sfc-gh-aivanou** [coredns/coredns#6028] [General]
Is there ways to enable plugins via kubeadm?
I want to start k8s cluster via `kubeadm init --config my_conf` .
Can I enable `log` plugin via kubeadm config file, so I don't need to manually enable it later via editing confipmag?

**Comment by johnbelamaric** [comment]
That would be a question for the `kubeadm` folks. It's outside the scope of this project.

I have thought about the idea of a Corefile parser that reads from K8s CRs, but it's a lot of work. It would be very nice though.
